test/SpeedTestSegReduce.chpl

```diff
@@ -0,0 +1,184 @@
+use TestBase;
```

Comment:
Just curious -- do you want to keep this test as-is for future tuning, or do you think it's served it's purpose with the initial experimentation?

Reply:
I think it's probably served its purpose. I'll move it to untested so it doesn't slow down the CI.